### PR TITLE
Forms: Response Inbox - Bulk Actions menu

### PR DESCRIPTION
**Pull request by jcheringer**
## Proposed changes:

* Replaces the bulk actions dropdown with a bulk actions menu

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Use the checkboxes to select responses 
- Play with the bulk actions on top of the table and check if the actions work properly

### CODE REVIEW

File: projects/packages/forms/src/dashboard/inbox/bulk-actions-menu.js [hunk 1]
```diff
@@ -1,34 +1,72 @@
...
-					<MenuItem icon={ archive } iconPosition="left">
-						{ __( 'Archive', 'jetpack-forms' ) }
-					</MenuItem>
-					<MenuItem icon={ trash } iconPosition="left">
-						{ __( 'Delete', 'jetpack-forms' ) }
-					</MenuItem>
-				</MenuGroup>
+		<>
+			{ currentView !== VIEWS.trash && (
+				<Button onClick={ onActionHandler( ACTIONS.moveToTrash ) } variant="secondary">
```
Comment (ice9js) [inline projects/packages/forms/src/dashboard/inbox/bulk-actions-menu.js:39]: Hmm, should we `disable` these buttons while `loading` is `true`? That should prevent any accidental race conditions with multiple requests going out simultaneously as well as providing some initial UI feedback to the user.

**Review by ice9js — CHANGES_REQUESTED** [review]
Looks good to me, let's maybe just disable the buttons while we're handling the requests and I think we should be all good 🙂.

**Review by digitalwaveride — APPROVED** [review]
Thanks João! Button interaction worked for me! 🚀 

We might need to do some styling finetuning on the buttons... but that will happen in a separate PR!

File: projects/packages/forms/src/dashboard/inbox/bulk-actions-menu.js (hunk 1 above)
Comment (jcheringer) [reply projects/packages/forms/src/dashboard/inbox/bulk-actions-menu.js:39]: The buttons are not visible while `loading` is `true`.
I'm hiding the Bulk Actions menu completely here: https://github.com/Automattic/jetpack/pull/29766/files#diff-f6dedbab4016eb9af19b412e0c0be98ae4342e84c9c69aa770554719c50983b7R80

Comment (ice9js) [reply projects/packages/forms/src/dashboard/inbox/bulk-actions-menu.js:39]: I guess that works too :)

**Review by ice9js — APPROVED** [review]

**Pull request merged**